### DISCUSSION TITLE
Trouble Scaffolding and Using DataConnections

### DISCUSSION THREAD

**Discussion by Jdillon297** [tareqimbasher/NetPad#289] [Q&A]
Hey there,  I love this product its awesome however I am running into issues with the omnisense being slow and if I type too fast then it never shows back up until I close the app and reopen it.  The bigger topic I wanted to ask about is I am able to add a data connection however it never seems to scaffold the database and I do not have access to my dbsets so I can use linq to query the tables.  Any information or syntax I may be missing would help.  I should also say on the SQL side I am able to query the table if it exists however I get no omnisense for this. Thanks.

**Comment by tareqimbasher** [comment]
Sorry for the late response, I must've missed this somehow. A newer version of NetPad was released that has alot of fixes. Give it a try, and if you still have issues I'd be happy to help. Glad you're enjoying NetPad!
